x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/PyTorchModelIT.java

```diff
@@ -244,10 +244,11 @@ public void testDeploymentStats() throws IOException {
             assertThat(byteSize, equalTo((int) RAW_MODEL_SIZE));
 
             Response humanResponse = client().performRequest(new Request("GET", "/_ml/trained_models/" + modelId + "/_stats?human"));
-            stats = (List<Map<String, Object>>) entityAsMap(humanResponse).get("trained_model_stats");
+            var responseMap = entityAsMap(humanResponse);
+            stats = (List<Map<String, Object>>) responseMap.get("trained_model_stats");
             assertThat(stats, hasSize(1));
             String stringBytes = (String) XContentMapValues.extractValue("deployment_stats.model_size", stats.get(0));
-            assertThat(stringBytes, is(not(nullValue())));
+            assertThat("stats response: " + responseMap, stringBytes, is(not(nullValue())));
```

Comment:
Then include `stats` from line 237 here as well as `responseMap`.

That way there will be more clues about what happened between the two calls to the same API that got inconsistent results.

For example, it could be that for some reason you get a _completely_ different response depending on which coordinating node you hit (which would obviously be a worse bug).